### PR TITLE
Big Hugh Mungus Gun Hoedown: The Gunnening

### PR DESCRIPTION
**Pull request by SeniorScore**
**SCOPE BUG IS GONE**
https://www.youtube.com/watch?v=-50NdPawLVY

Alright so this fucking thing's m a s s i v e. Changelog anyway because it's actually pretty fucking important.

Primary goal of this update is to introduce the variables that can make certain guns do more damage/AP without the ammo being a limiting factor. This increases flexibility in weapons that use the same ammo that was not achievable before hand. So more weapon variance will be a large goal of this update. As as a result, a lot of base ammo _was_ nerfed, but again there's variables at play in the guns now.

Also added Mosins to spawn in garbage piles. They're shitty hunting rifles now why not.

**SHOTGUNS ARE UNTOUCHED BY MOST OF THESE CHANGES BECAUSE BUCKSHOT IS SOME AUTISTIC SHIT WITH INCREASED DAMAGE VARIABLES**

tl;dr War has changed.

<--AMMO-->

REMOVED: 50 AP, 45 LC (they're both meme rounds in different ways)

9mm(unchanged) - 20, 0
45 - 25, 0
10mm - 20, 0
.44 Magnum - 30, 0
.357 Magnum - 30, 0
38 Special(unchanged) - 25, 0
45-70 - 45, 0
5.56 - 20, 0
50 BMG - 60, 20

<-- WEAPONS -->

This is a lot bigger, in terms of balance.. and actual, ya know, changelog size. Added fire delays onto a lot of weapons in addition to the extra damage and penetration variables.

We'll start with 

**Pistols**

9mm Pistol - 25 dmg, 0 AP, fire delay of 2
10mm Pistol - 25 , 5. 2
M1911 - 30, 0, 2
.357 Magnum Revolver - 30, 0, 5
.44 Magnum Revolver - 30, 10, 3
Scoped .44 Magnum Revolver - 30, 10, 4
Colt 650 - 30, 10, 3
Ranger Sequoia - 45, 0, 4
Scoped Hunting Revolver - 45, 0, 4
Zipgun - 40, 0, no fire delay (should probably changed that actually)

**SMGS**

9mm SMG - 20 dmg, 10 AP, fire delay of 2
Uzi -20, 10, 1
10mm SMG -20, 15, 1
Thompson SMG -25, 0, 3

**Rifles**
Hunting Rifle (unchanged, honestly still one of the most balanced weapons in game) - 40 dmg, 10 AP (with 7.62), no fire delay
Scoped Hunting Rifle - 40, 10, 3
Mosin - 40, 10, 5
Colt Rangemaster - 40, 10, 3
R91 Assault Rifle - 20, 20, 4
R81 Service Rifle - 25, 5, 3
R91 Infiltrator - 20, 10, 1
R94 Marksman - 40, 0, 8
Bozar - 35, 15, 3
Gauss Rifle - 60, 40, 10
Cowboy Repeater - 40, 0, 7
Scoped Cowboy Repeater -40, 10, 8
Trail Carbine - 40, 10, 4
Scoped Trail carbine - 45, 10, 7
Brushgun - 55. 10, 5
Scoped Brushgun -60, 10, 9
AMR - 60, 20, 6
Pipe Rifle (memes inbound) - 50, 0, 0

### CODE REVIEW

**Comment by AdamElTablawy** [comment]
given the firedelay, laser weapons are suddenly going to be vastly preferable to basically every other weapon out there.

also the firedelay on some of these makes them almost unusable.

**Comment by SeniorScore** [comment]
I think the one's with substantial fire delay certainly earn it. As for lasers, you're right, their absence of fire-delays, and my looking of how much damage they're actually fucking doing. Given them fire-delays, usually around 3, removed that eye blur because just., why, and knocked plasma down to 50 for the caster, and 45 for the pistol. Still respectable numbers but not making me want to fucking scream.

**Comment by SeniorScore** [comment]
SCOPE BUG IS GONE

**Comment by SeniorScore** [comment]
Nerfed Shotgun slugs after Adam told me they had been adjusted to be 50/20, which would make them... powerful compared to rifles in this update. They are now just 50 damage. Went through and added fire delays to them as well, and the hunting rifle I believe, of 2.
